ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/stores/ledger/ImplicitPartyAdditionIT.scala

```diff
@@ -95,44 +70,6 @@ class ImplicitPartyAdditionIT
     }
   }
 
-  private def publishSingleNodeTx(
-      ledger: Ledger,
-      submitter: String,
-      commandId: String,
-      node: Transaction.AbsNode,
-  ): Future[SubmissionResult] = {
-    val event1: Transaction.NodeId = Transaction.NodeId(0)
-
-    val let = Time.Timestamp.assertFromInstant(LET)
-
-    val transaction: Transaction.AbsTransaction = GenTransaction(
-      HashMap(event1 -> node),
-      ImmArray(event1),
-    )
-
-    val submitterInfo = SubmitterInfo(
-      Ref.Party.assertFromString(submitter),
-      Ref.LedgerString.assertFromString("appId"),
-      Ref.LedgerString.assertFromString(commandId),
-      Time.Timestamp.assertFromInstant(MRT),
-      DeduplicateUntil,
-    )
-
-    val transactionMeta = TransactionMeta(
-      ledgerEffectiveTime = let,
-      workflowId = Some(Ref.LedgerString.assertFromString("wfid")),
-      submissionTime = let.addMicros(1000),
-      submissionSeed = None,
-      optUsedPackages = None,
-    )
-
-    ledger.publishTransaction(submitterInfo, transactionMeta, transaction)
-  }
-
-  val LET = Instant.EPOCH.plusSeconds(10)
-  val MRT = Instant.EPOCH.plusSeconds(10)
-  val DeduplicateUntil = Instant.now.plusSeconds(3600)
```

Comment:
IntelliJ likes to complain when you don't add type annotations to non-private variables. These three caused warnings, which was annoying.